src/genie/libs/parser/iosxe/tests/c9300/test_show_platform.py

```diff
@@ -155,6 +156,717 @@ def test_show_inventory_c9300(self):
         parsed_output = obj.parse()
         self.assertEqual(parsed_output, self.golden_parsed_output)
 
+class TestShowPlatformTcamUtilization(unittest.TestCase):
+    maxDiff = None
+    device_empty = Device(name='empty')
+    device = Device(name='9300-FE3')
+    empty_output = {'execute.return_value': ''}
+
+
+    # show inventory
+    golden_output = {'execute.return_value': '''
+Codes: EM - Exact_Match, I - Input, O - Output, IO - Input & Output, NA - Not Applicable

```

Comment:
Add extra space to align with golden_output

Reply:
done and resolve